math/shift_of_sampling_points_of_polynomial/gen/N_1.cpp

```diff
@@ -0,0 +1,24 @@
+#include <cstdlib>
+#include <cstdio>
+
+#include "../params.h"
+#include "random.h"
+
+int main(int, char* argv[]) {
+  long long seed = std::atoll(argv[1]);
+  auto gen = Random(seed);
+
+  int n = 1, m = gen.uniform<int>(1, N_MAX), c = gen.uniform<int>(0, MOD - 1);
```

Comment:
> Since it seems OK for now (as the checks pass), this is just a note: I am not a C++ expert, but I wonder if the evaluation order of these initializations is specified in all compilers (though I think it is usually evaluated from left to right).

Thanks for reviewing! I wrote this before I know the the evaluation order. Sorry for my carelessness. Fortunately, it seems to be fine. I found some descriptions in this [page](https://en.cppreference.com/w/cpp/language/declarations), it seems that `S D1, D2;` is same as `S D1; S D2;`.